sentry-android-core/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools">
   <uses-permission android:name="android.permission.INTERNET"/>
+
+  <!-- Compose requires min SDK 21, but Sentry requires min SDK 19 -->
+  <uses-sdk tools:overrideLibrary="androidx.compose.runtime" />
```

Comment:
is this needed only for tests/sample apps?

Reply:
Ah yeah, not needed anymore :+1: